src/components/seed-search-form.jsx

```diff
@@ -197,7 +197,7 @@ class SeedSearchForm extends React.Component {
         <h3 className="title is-3">Step 1: Seed your playlist</h3>
         <div className="control">
           <label className="label" htmlFor="seed">
-            Find songs like:
+            Find {seedType}s like:
```

Comment:
Hmm, I think this isn't right, actually: whether you choose an artist seed or a track seed, the ["get recommendations" endpoint](https://developer.spotify.com/web-api/get-recommendations/) will return tracks. So "Find songs" is always accurate. 🤔